spec/rubocop/server/cli_spec.rb

```diff
@@ -145,6 +145,24 @@
         expect($stderr.string).to eq "--server-status cannot be combined with other options.\n"
       end
     end
+
+    context 'when using server option with `--server-root path` option' do
+      it 'returns exit status 0 and display an error message' do
+        expect(cli.run(['--server-status', '--server-root /tmp'])).to eq(0)
+        expect(cli.exit?).to be(true)
+        expect($stdout.string).to eq "RuboCop server is not running.\n"
+        expect($stderr.string).not_to eq "--server-status cannot be combined with other options.\n"
+      end
+    end
+
+    context 'when using server option with `--server-root=path` option' do
+      it 'returns exit status 0 and display an information message' do
+        expect(cli.run(['--server-status', '--server-root=/tmp'])).to eq(0)
+        expect(cli.exit?).to be(true)
+        expect($stdout.string).to eq "RuboCop server is not running.\n"
+        expect($stderr.string).not_to eq "--server-status cannot be combined with other options.\n"
+      end
+    end
```

Comment:
Everywhere you have `server-root` here, should it be `cache-root`? Or is this on purpose?

Reply:
Oops! I've updated to `--cache-root` and adjusted the test code a little.